x/concentrated-liquidity/types/constants.go

```diff
@@ -6,8 +6,8 @@ import (
 
 // TODO: decide on the values for Max tick and Min tick
 var (
-	MinTick int64 = -887272
-	MaxTick int64 = 887272
+	MinTick int64 = -9999887272
+	MaxTick int64 = 9999887272
```

Comment:
How were these chosen?

Reply:
Arbitrarily. The TODO above it is that we need to figure out what we should set these to.

Reply:
I just needed to expand this from what it was before since the number of ticks we need to be able to init vastly expanded